zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -428,6 +428,7 @@ public String getConfDir() {
     // Decide when new note is created, interpreter settings will be binded automatically or not.
     ZEPPELIN_NOTEBOOK_AUTO_INTERPRETER_BINDING("zeppelin.notebook.autoInterpreterBinding", true),
     ZEPPELIN_CONF_DIR("zeppelin.conf.dir", "conf"),
+    ZEPPELIN_INTERPRETER_REPO_DIR("zeppelin.user.interpreter.dir", "interpreter"),
```

Comment:
I think the name `ZEPPELIN_INTERPRETER_REPO_DIR` and `zeppelin.user.interpreter.dir` are little bit confusing.

How about more intuitive name such as `ZEPPELIN_INTERPRETER_DOWNLOAD_DIR` and `zeppelin.interpreter.download.dir` ?